src/renderer/components/Editor/TakeGroupComponent.tsx

```diff
@@ -95,8 +100,15 @@ const TakeGroupComponent = ({
     handleModalClose();
   };
 
-  const clickAway = () => {
-    if (!isFirstTimeOpen) {
+  const onClickAway = (event: Event) => {
+    const isInTranscriptionContent = isEventInElement(
+      event,
+      transcriptionContentId
+    );
+    const isInTakeGroup = isEventInElement(event, takeGroupId);
```

Comment:
What will happen here if the user clicks away from the current take group onto another take group? Do the id's need to be unique?

Reply:
The ID's dont need to be unique. Clicking on another take group will simply open it

Reply:
Is that the behaviour we want? Shouldn't clicking on another take group close the current one?

Reply:
Yeah @exquisitebin is assigned to "Clicking away from take group closes take group and chooses currently selected take" - so clicking another take group should close the current one

Reply:
Emma is correct. Clicking on another Take Group closes the currently opened one. That functionality is in develop.